include/crocoddyl/core/activations/quadratic-flat.hpp

```diff
@@ -0,0 +1,155 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020, LAAS-CNRS
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_FLAT_HPP_
+#define CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_FLAT_HPP_
+
+#include "crocoddyl/core/activation-base.hpp"
+#include "crocoddyl/core/fwd.hpp"
+#include "crocoddyl/core/utils/exception.hpp"
+#include <iostream>
+#include <stdexcept>
+
+namespace crocoddyl {
+
+/**
+ * @brief Quadratic-flat activation
+ *
+ * This activation function describes a quadratic exponential activation for
+ * each element of a residual vector, i.e. \f[ \begin{equation} sum^nr_{i=0} 1 -
+ * exp(\|r_i\|^2 / \alpha) \end{equation} \f] where \f$\alpha\f$ defines the
```

Comment:
If I understand correctly, this is not an element-wise operation. If so, you should write:
```cpp
\begin{equation} 1 - exp(\|\mathbf{r}\|^2 / \alpha \end{equation}
```